p4-16/psa/PSA.mdk

```diff
@@ -1,5 +1,5 @@
 Title : P4~16~ Portable Switch Architecture (PSA)
-Title Note : (working draft)
+Title Note: Version 1.0
 Title Footer: &date;
```

Comment:
Do we want to hard-code a particular date like 2018-03-01 in place of `&date;` here, too?  (then change it back after the 1.0 release, as these other changes likely will be).

Reply:
Except for that, this looks good to me, and I would recommend please that you merge it in for the release.